lib/usd/pxrUsdPreviewSurface/AEusdPreviewSurfaceTemplate.mel

```diff
@@ -31,7 +31,7 @@ global proc pxrUsdPreviewSurface_ToggleSpecularWorkflow(string $nodeName)
     }
 }
 
-global proc AEpxrUsdPreviewSurfaceTemplate(string $nodeName)
+global proc AE${PREVIEWSURFACE_MAYA_TYPE_NAME}Template(string $nodeName)
```

Comment:
File is back where it belongs. It now has template parameters allowing clients to generate a copy that works with their own node.